### DISCUSSION TITLE
My GitHub Sponsor account is still pending for 2 weeks.

### DISCUSSION THREAD

**Discussion by skyrsg** [community/community#137810] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello folks?

I am working on one open-source project.
And there is someone who wants to sponsor me, so he asked me tie github sponsor acc to my github profile.

So I tried to do it last Monday, but it's still on pending status.

Is there anyone who knows the reason?

If so, plz let me know.

Thank you

**Comment by davevad93** [comment, marked as answer]
Hi @rdcruzer0315 ,

GitHub may take a few days to review your request, wait until next week and if by then your status is still pending, try contacting support by opening a ticket [here](https://support.github.com/contact/account), select `GitHub Sponsors` and explain your issue.

**Reply by skyrsg** [reply]
thank you. @davevad93

**Reply by davevad93** [reply]
You're welcome @rdcruzer0315 👍

**Comment by ebndev** [comment]
Hi @rdcruzer0315!

In addition to opening a ticket on our [Support page](https://support.github.com/contact), you can also check out the [Sponsors FAQ](https://github.com/orgs/community/discussions/116811) and see if you there's any information that may be relevant to you.

**Comment by hhuayuan** [comment]
My sponsor still on pending status for few months, I don't know what can i do.
